### ISSUE TITLE
Shuttle stops working after upgrade to  Big Sur

### ISSUE THREAD

**Issue by seaskyv** [fitztrev/shuttle#272]
This is a fantastic app for MacOS especially for people who need frequently ssh with iterm to multiple hosts.
But since recently upgrade MacOS to Big Sur, it stops works. Icon can be launched in top bar but, stop working, and also cause external MS keyboard stop working. 

Hopefully someone can upgrade it to compatible with the new MacOS.

**Comment by liuzdcq** [comment]
System Preferences -> Security & Privacy -> Privacy -> Automation -> Check on shuttle

**Comment by holywen** [comment]
It can open only the first terminal tab after updated the preferences. nothing happens when I try to open the second connection in new terminal tab.

**Comment by liuzdcq** [comment]
> It can open only the first terminal tab after updated the preferences. nothing happens when I try to open the second connection in new terminal tab.

That is true, I didn't notice that before. The simple solution is to use 'new' instead of 'tab'.

**Comment by ChenRenHu** [comment]
> > It can open only the first terminal tab after updated the preferences. nothing happens when I try to open the second connection in new terminal tab.
> 
> That is true, I didn't notice that before. The simple solution is to use 'new' instead of 'tab'.

I have the same problem.

**Comment by ChenRenHu** [comment]
> > > It can open only the first terminal tab after updated the preferences. nothing happens when I try to open the second connection in new terminal tab.
> > 
> > 
> > That is true, I didn't notice that before. The simple solution is to use 'new' instead of 'tab'.
> 
> I have the same problem.



It seems I had solved the problem.  it seems the problem of terminal. 

System Preferences -> Security & Privacy -> Privacy -> Assits  -> Check on terminal;

**Comment by zixiliuyue** [comment]
![image](https://user-images.githubusercontent.com/12575317/100321973-7ff6a000-2ffe-11eb-9ef6-82628b2585cd.png)
Doesn't work

**Comment by holywen** [comment]
> > > > It can open only the first terminal tab after updated the preferences. nothing happens when I try to open the second connection in new terminal tab.
> > > 
> > > 
> > > That is true, I didn't notice that before. The simple solution is to use 'new' instead of 'tab'.
> > 
> > 
> > I have the same problem.
> 
> It seems I had solved the problem. it seems the problem of terminal.
> 
> System Preferences -> Security & Privacy -> Privacy -> Assits -> Check on terminal;

I cannot find this "Assits" , but FYI, I found iTerm2 works fine to create new tab.
hope this helps.

**Comment by dengblom** [comment]
I changed my settings to 
"terminal": "Terminal.app",
"open_in": "new",

and that seems to work for me.  It opens in a new tab, rather than a new window, but that's what I would prefer anyway.

**Comment by peterbuga** [comment]
the comments related to privacy settings helped for me: quit Shuttle, manually remove any old reference to Shuttle in Security & Privacy, then re-add them manually, start the app and accept any other prompts you might get.

hope it helps someone 🤷‍♂️

**Comment by ccmuhub** [comment]
> the comments related to privacy settings helped for me: quit Shuttle, manually remove any old reference to Shuttle in Security & Privacy, then re-add them manually, start the app and accept any other prompts you might get.
> 
> hope it helps someone 🤷‍♂️

Worked for me - thanks!
